aws/endpoints/sts_legacy_regions.go

```diff
@@ -5,7 +5,6 @@ var stsLegacyGlobalRegions = map[string]struct{}{
 	"ap-south-1":     {},
 	"ap-southeast-1": {},
 	"ap-southeast-2": {},
-	"aws-global":     {},
```

Comment:
Should this have been removed? We still set the region to "aws-global" in v3model.go#L115

Reply:
Yes this was intentional. The `aws-global` region already exists in the model, and wasn't really a legacy region. If endpoint were resolved with this region it was resolve to aws-global direclty in the endpoints file.